onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4736,18 +4736,18 @@ def aten_to_sparse_csr(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
+@torch_op("aten::topk", trace_only=True)
 def aten_topk(
-    self: TensorType, k: int, dim: int = -1, largest: bool = True, sorted: bool = True
-) -> tuple[TensorType, TensorType]:
+    self: TReal, k: INT64, dim: int = -1, largest: bool = True, sorted: bool = True
+) -> Tuple[TReal, INT64]:
```

Comment:
```suggestion
) -> tuple[TReal, INT64]:
```

nit: tuple is good since we imported the future

Reply:
got this error when it was lower case tuple
```
______________________________________________________________________________________________________ ERROR collecting onnxscript/test/function_libs/torch_aten/ops_correctness_test.py ______________________________________________________________________________________________________
onnxscript/test/function_libs/torch_aten/ops_correctness_test.py:18: in <module>
    from onnxscript.function_libs.torch_aten.ops import core as core_ops
onnxscript/function_libs/torch_aten/ops/core.py:4758: in <module>
    def aten_topk(
onnxscript/function_libs/torch_aten/registration.py:74: in wrapper
    processed_func = onnxscript.script()(func)
onnxscript/main.py:107: in transform
    result = script_check(ast, opset, env, src, default_opset=default_opset)
onnxscript/main.py:53: in script_check
    return convert.top_level_stmt(f)
onnxscript/converter.py:1357: in top_level_stmt
    fn_ir = self.translate_function_def(stmt)
onnxscript/converter.py:1331: in translate_function_def
    type_annotation = self.eval_constant_expr(fn.returns)
onnxscript/converter.py:412: in eval_constant_expr
    return eval(cpl, self.globals, locals)  # noqa: DUO104
<ast>:4: in <module>
    ???
E   TypeError: 'type' object is not subscriptable
=================================================================================================================================== short test summary info ===================================================================================================================================
ERROR onnxscript/test/function_libs/torch_aten/ops_correctness_test.py - TypeError: 'type' object is not subscriptable
```

Reply:
Ahh ok I see what's going on. Tuple is good then